bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -622,7 +633,9 @@ private void finalizeTour() {
                tourData.setPower_AvgRightPedalSmoothness(val);
                _currentActivity._avgPowerRightPedalSmooth = val;
             } catch (final Exception exc) {
-               StatusUtil.log(CONST_RIGHT_PEDAL_SMOOTH + ": FitlogEx parse error", exc); //$NON-NLS-1$
+
+               StatusUtil.log(CONST_RIGHT_PEDAL_SMOOTH + ": " + CONST_FITLOGEX_ERROR_STRING, exc); //$NON-NLS-1$
+
             }
          }
       }
```

Comment:
Error in line 799, could not set the comment on the correct line

![mt-error](https://user-images.githubusercontent.com/1283445/104458802-20ebeb00-55ac-11eb-9256-bcf834f3a6ea.png)

Reply:
@wolfgang-ch , I don't have that error at all, it is in the function "private void finalizeTour()" 
that last "}" ending that function is correct and doesn't have any issue locally for me. So it is most likely due to merge.